gazelle/internal/spdump/dependency.go

```diff
@@ -59,19 +60,67 @@ func (sc *SourceControl) UnmarshalJSON(b []byte) error {
 	return nil
 }
 
+type sourceControlLocationXcode15 struct {
+	Remote []*RemoteLocation
+}
+
+type sourceControlLocationXcode14 struct {
+	Remote *RemoteLocation
+}
+
 // A SourceControlLocation represents the location of a source control repository.
 type SourceControlLocation struct {
 	Remote *RemoteLocation
 }
 
+func (scl *SourceControlLocation) UnmarshalJSON(b []byte) error {
+
+	// Try the Xcode 15 format first:
+	var sclx15 sourceControlLocationXcode15
```

Comment:
Instead of using Xcode versions, could we use Swift versions? That will make it clearer for anyone working on non-Apple systems.

Reply:
Ok I changed the names to refer to Swift 5.9 / 5.8 instead.